osidb/tests/test_audit.py

```diff
@@ -0,0 +1,176 @@
+import uuid
+
+import pghistory
+import pytest
+from django.db import transaction
+from django.utils import timezone
+
+from osidb.core import set_user_acls
+from osidb.models import Affect, Flaw, FlawMeta, FlawSource, Impact
+from osidb.tests.factories import AffectFactory, FlawFactory, FlawMetaFactory
+
+pytestmark = pytest.mark.unit
+
+
+@pytest.mark.django_db(transaction=True)
+class TestAuditFlaw:
+    @property
+    def acl_read(self):
+        return [
+            uuid.uuid5(
+                uuid.NAMESPACE_URL,
+                "https://osidb.prod.redhat.com/ns/acls#data-prodsec",
+            )
+        ]
+
+    @property
+    def acl_write(self):
+        return [
+            uuid.uuid5(
+                uuid.NAMESPACE_URL,
+                "https://osidb.prod.redhat.com/ns/acls#data-prodsec-write",
+            )
+        ]
+
+    def test_flawevent(self):
+        """ """
+        # create a flaw with an affect
+        Flaw.objects.all().delete()
```

Comment:
I wonder why the flaws are deleted first. Looking at the event count later it seems that it generates no event so I guess it has no effect - which is what I would assume at the beginning of a test (no flaws present).